Documentation/ceph-filesystem-crd.md

```diff
@@ -124,6 +198,17 @@ The metadata server settings correspond to the MDS daemon settings.
 
 * `activeCount`: The number of active MDS instances. As load increases, CephFS will automatically partition the filesystem across the MDS instances. Rook will create double the number of MDS instances as requested by the active count. The extra instances will be in standby mode for failover.
 * `activeStandby`: If true, the extra MDS instances will be in active standby mode and will keep a warm cache of the filesystem metadata for faster failover. The instances will be assigned by CephFS in failover pairs. If false, the extra MDS instances will all be on passive standby mode and will not maintain a warm cache of the metadata.
+* `mirroring`: Sets up mirroring of the filesystem
+  * `enabled`: whether mirroring is enabled on that filesystem (default: false)
+  * `peers`: to configure mirroring peers
+    * `secretNames`:  a list of peers to connect to. Currently (Ceph Octopus release) **only a single** peer is supported where a peer represents a Ceph cluster.
+  * `snapshotSchedules`: schedule(s) snapshot.One or more schedules are supported.
+    * `path`: filesystem path to take the snapshot on
```

Comment:
Instead of renaming the Path to SourcePath in the code, how about this?
```suggestion
    * `path`: filesystem source path to take the snapshot on
```